src/daft-schema/src/schema.rs

```diff
@@ -391,6 +391,42 @@ impl Schema {
         }))
         .to_string()
     }
+
+    pub fn max_fixed_byte_column(&self) -> Option<&str> {
+        self.fields
+            .iter()
+            .filter(|f| f.dtype.is_fixed_size())
+            .filter_map(|f| f.fixed_byte_size().map(|s| (s, f.name.as_str())))
+            .fold(None, |max, (size, name)| match max {
+                Some((max_size, max_name)) => {
+                    if size > max_size {
+                        Some((size, name))
+                    } else {
+                        Some((max_size, max_name))
+                    }
+                }
+                None => Some((size, name)),
+            })
+            .map(|(_, name)| name)
+    }
+
+    pub fn min_fixed_byte_column(&self) -> Option<&str> {
+        self.fields
+            .iter()
+            .filter(|f| f.dtype.is_fixed_size())
+            .filter_map(|f| f.fixed_byte_size().map(|s| (s, f.name.as_str())))
+            .fold(None, |min, (size, name)| match min {
+                Some((min_size, min_name)) => {
+                    if size < min_size {
+                        Some((size, name))
+                    } else {
+                        Some((min_size, min_name))
+                    }
+                }
+                None => Some((size, name)),
+            })
+            .map(|(_, name)| name)
+    }
```

Comment:
I think we don't necessarily need to filter for the minimum _fixed size_ column, and can just use the estimates to select the smallest. Even if there are only variable sized columns we can still pick the column we estimate to be the smallest (for example one column might be a struct with a single string field while another could be a struct with many string fields).